docs/README.md

```diff
@@ -1044,6 +1044,140 @@ docker-compose up
 2. Site will be avaiable on `localhost:3001`
 3. You can update the documentation by modifying the file `docs/README.md`
 
+## OpReturn
+
+OP_RETURN is a [script opcode](https://wiki.bitcoinsv.io/index.php/OP_RETURN) that allows adding extra information to transactions beyond standard inputs and outputs. This can be useful for applications that want to include:
+
+* Different transaction types related to events (e.g., gaming);
+
+* Invoice numbers;
+
+* Nonces to differentiate payments of the same amount;
+
+* Other data.
+
+### General OpReturn structure
+
+    <OP_RETURN opcode> <pushdata for protocol identifier> <protocol identifier> <version> <pushdata for data payload> <data payload> <pushdata for nonce> <nonce>
+
+### General Syntax Rules:
+
+The data following the OP_RETURN is composed by (n) mandatory components:
+
+1. **OP_RETURN opcode**: 6a
+
+2. Pushdata that will indicate the size of the protocol identifier (in bytes)
+
+3. **Protocol identifier**: `50415900` for Paybutton (ASCII `PAY` + `0x00`)
+
+4. **Version**: A byte allowing future updates (currently `0`).
+
+5. Pushdata that will indicate the size of data payload identifier (in bytes)
+
+6.  **Data Payload**: Custom information in UTF-8 format with the indicated size (can be empty).
+
+7. Pushdata that will indicate the size of nonce identifier (in bytes)
+
+8.  **Nonce**: Eight random bytes to differentiate payments (can be empty).
+
+If the **data payload** or **nonce** is empty, the pushdata of each will be 00 
+
+
+  
+### How to send data using OP_RETURN opcode in Paybutton
+
+To send data using the OP_RETURN opcode in Paybutton, you can use the `opReturn` prop. The content of the `opReturn` prop will be encoded and will correspond to the **data payload**, mentioned above. Additionally, you may use the **payment ID** as a nonce. To disable sending the payment ID, use the `disablePaymentID` prop — Paybutton will automatically encode the message according to the rules specified bellow.
+
+
+### Paybutton OpReturn encode examples:
+
+
+#### 1. OpReturn message with 12 bytes of data and no nonce
+  
+
+    6a0450415900000c0102030405060708090a0b0c00
+
+  
+
+Breaking this down:
+
+  
+
+-  `6a` → OP_RETURN opcode
+
+-  `04` → Pushdata indicating the size (in bytes) of the data protocol identifier
+
+-  `50415900` → PayButton identifier (ASCII `PAY` + `0x00`)
+
+-  `00` → Version 0
+
+-  `0c` → Pushdata for the data payload, indicating 12 bytes of data.
```

Comment:
dot at the end of the sentence here but not in the other breakdown lines; just choose one way (with or without) and keep it consistent.

Reply:
Same thing about the dot in the other examples in this page